src/main/java/usersandpostsCRUD/demo/entity/Post.java

```diff
@@ -0,0 +1,76 @@
+package usersandpostsCRUD.demo.entity;
+
+
+import jakarta.persistence.*;
+import org.hibernate.annotations.ManyToAny;
+
+import java.time.LocalDateTime;
+@Entity
+public class Post {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+    private String description;
+    @ManyToOne
+    @JoinColumn(name = "user_id",nullable = false)
+    private User user;
```

Comment:
What is the difference between unidirectional and bidirectional mapping?
Explan in a reply comment

Reply:
In unidirectional mapping u can only access the relationship from the parent side which manages the foreign key.In bidirectional mapping u can navigate the relationship in both ways